mlir/lib/Dialect/Linalg/IR/LinalgInterfaces.cpp

```diff
@@ -58,8 +58,8 @@ bool linalg::detail::canOpOperandsBeDroppedImpl(
 // CopyOpInterface implementation
 //===----------------------------------------------------------------------===//
 
-bool linalg::isaCopyOpInterface(LinalgOp op) {
-  // Check all loops are parallel and linalgOp is single input and output.
+bool linalg::isaCopyOpInterface(GenericOp op) {
+  // Check all loops are parallel and genericOp is single input and output.
```

Comment:
Why constrain it to `generic` only?

It's convenient to have one general utility for verifying copy-like behavior.